distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestCatalog.java

```diff
@@ -136,6 +140,76 @@ public void testCswIngest() {
+            ValidatableResponse response = executeOpenSearch("xml", "q=*");
+            response.body(hasXPath("/metacards/metacard[@id='" + id1 + "']"));
+
+            startFeature(true, "sample-filter");
+            startFeature(true, "filter-plugin");
+
+            // Configure the PDP
+            PdpProperties pdpProperties = new PdpProperties();
+            pdpProperties.put("matchAllMappings",
+                    "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role=point-of-contact");
+            Configuration config = configAdmin
+                    .getConfiguration("ddf.security.pdp.realm.SimpleAuthzRealm", null);
+            Dictionary<String, ?> configProps = new Hashtable<>(pdpProperties);
+            config.update(configProps);
+            waitForAllBundles();
+
+            // Test with filtering with out point-of-contact
+            response = executeOpenSearch("xml", "q=*");
+            response.body(not(hasXPath("/metacards/metacard[@id='" + id1 + "']")));
+
+            // Test filtering with point of contact
+            configureRestForBasic();
+
+            response = executeAdminOpenSearch("xml", "q=*");
+            response.body(hasXPath("/metacards/metacard[@id='" + id1 + "']"));
+
+            configureRestForAnon();
+
+            stopFeature(true, "sample-filter");
+            stopFeature(true, "filter-plugin");
+
+            deleteMetacard(id1);
+        } catch (Exception e) {
+            LOGGER.error("Couldn't start filter plugin");
+        }
+    }
+
+    private void configureRestForBasic() throws IOException, InterruptedException {
+        PolicyProperties policyProperties = new PolicyProperties();
+        policyProperties.put("authenticationTypes",
+                "/=SAML|basic,/admin=SAML|basic,/jolokia=SAML|basic,/system=SAML|basic,/solr=SAML|PKI|basic");
+        policyProperties.put("whiteListContexts",
+                "/services/SecurityTokenService,/services/internal,/proxy," + SERVICE_ROOT
+                        + "/sdk/SoapService");
+        Configuration config = configAdmin.getConfiguration(PolicyProperties.FACTORY_PID, null);
+        Dictionary<String, ?> configProps = new Hashtable<>(policyProperties);
+        config.update(configProps);
+        waitForAllBundles();
+    }
+
+    private void configureRestForAnon() throws IOException, InterruptedException {
+        PolicyProperties policyProperties = new PolicyProperties();
+        policyProperties.put("authenticationTypes",
+                "/=SAML|anon,/admin=SAML|anon,/jolokia=SAML|anon,/system=SAML|anon,/solr=SAML|PKI|anon");
+        policyProperties.put("whiteListContexts",
+                "/services/SecurityTokenService,/services/internal,/proxy," + SERVICE_ROOT
+                        + "/sdk/SoapService");
+        Configuration config = configAdmin.getConfiguration(PolicyProperties.FACTORY_PID, null);
+        Dictionary<String, ?> configProps = new Hashtable<>(policyProperties);
```

Comment:
Why are these properties extensions of `HashMap` but using `Hashtable` for the implementation here?